### DISCUSSION TITLE
Calificacion Actividad S3P2

### DISCUSSION THREAD

**Discussion by gonaiot** [ISPC-TST-Electronica-Microcontrolada/Grupo8#44] [Announcements]
Calificacion Grupal: 8

@ProgramadorPegasus : 9
@tecnosisnet : 9
@MaviSchaf : 7
@evera8 : 7
@Cabro645 : 7

Observacion: Chicos trabajaron muy bien, la verdad que se ve uniformidad es su trabajo, los alumnos que no hacen commit, traten de capitalizar su aporte. que quede plasmado en el repositorio. Pero en general trabajaron muy bien como equipo, sigan asi.

**Comment by tecnosisnet** [comment]
Gracias profe !

**Reply by evera8** [reply]
Buenos dias, Entendido profe, muchas gracias.

**Reply by Cabro645** [reply]
Gracias Profe

**Comment by MaviSchaf** [comment]
Muchas gracias profe!
